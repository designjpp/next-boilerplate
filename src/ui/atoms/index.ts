export * from './button';
export * from './mouse';
export * from './scrollbar';
export * from './wow-title';
