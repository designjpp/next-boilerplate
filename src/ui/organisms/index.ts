// Put your organisms here
export const Organism = () => null;
