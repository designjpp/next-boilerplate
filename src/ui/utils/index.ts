export * from './media-queries';
export * from './tailwind-config';
