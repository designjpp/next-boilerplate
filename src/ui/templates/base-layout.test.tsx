import { create } from 'react-test-renderer';

import { BaseLayout } from 'ui';

describe('<BaseLayout>', () => {
	it("should match is's snapshot", () => {
		const component = create(<BaseLayout />).toJSON();
		expect(component).toMatchSnapshot();
	});
});
