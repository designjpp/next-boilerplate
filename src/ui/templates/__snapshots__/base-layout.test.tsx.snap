// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BaseLayout> should match is's snapshot 1`] = `
Array [
  <nav
    className="jsx-2513862931"
  >
    <a
      className="jsx-2513862931"
      href="/"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      Go to home
    </a>
    <a
      className="jsx-2513862931"
      href="/pets"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      See Pets!
    </a>
    <a
      className="jsx-2513862931"
      href="/react-query/client-side"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      Go to /react-query
    </a>
    <a
      className="jsx-2513862931"
      href="/react-query/static-props"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      Go to /react-query/static-props
    </a>
  </nav>,
  <main
    className="jsx-2513862931 "
  />,
]
`;
